chart/templates/_jobs.tpl

```diff
@@ -137,31 +137,49 @@
 
 {{- /*
 ==========================================================================================
-| _jobs.move $.Values.jobs $from_ig $to_ig $job
+| _jobs.move $ $from_ig_name $to_ig_name $job_name
 +-----------------------------------------------------------------------------------------
 | Move a job from one instance group to another, creating the destination instance
-| group if it doesn't exist yet. The new instance group will *not* inherit the default
-| condition from the old instance group.
+| group if it doesn't exist yet.
 ==========================================================================================
 */}}
 {{- define "_jobs.move" }}
-  {{- $jobs := index . 0 }}
-  {{- $from_ig := index . 1 }}
-  {{- $to_ig := index . 2 }}
-  {{- $job := $to_ig }}
-  {{- if gt (len .) 3 }}
-    {{- $job = index . 3 }}
+  {{- $root := index . 0 }}
+  {{- $from_ig_name := index . 1 }}
+  {{- $to_ig_name := index . 2 }}
+  {{- $job_name := index . 3 }}
+
+  {{- /* Locate $from_ig in the manifest */}}
+  {{- $_ := include "_config.lookupManifest" (list $root "instance_groups" $from_ig_name) }}
+  {{- $from_ig := $root.kubecf.retval }}
+  {{- if not $from_ig }}
+    {{- include "_config.fail" (printf "Instance group %q does not exist" $from_ig_name) }}
```

Comment:
Would a more detailed error be useful?

```suggestion
    {{- include "_config.fail" (printf "Could not find instance group %q while moving job %q" $from_ig_name $job_name) }}
```

(This depends on if we get a useful stack trace when things go sideways.)

Reply:
We do get a full stacktrace from `_config.fail`, so it doesn't really matter. Since users are not expected (anymore) to even attempt to move jobs around, these errors also should only happen during development.

I changed this error message anyways, but left the other ones alone.